tests/test_project/tests/utils.py

```diff
@@ -0,0 +1,179 @@
+import json
+import os
+
+from django.contrib.auth import get_user_model
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+
+User = get_user_model()
+
+
+class TestConfig(object):
+    """
+    Get the configurations that are to be used for all the tests.
+    """
+
+    config_file = os.path.join(os.path.dirname(__file__), 'config.json')
+    root_location = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..')
+    with open(config_file) as json_file:
+        config = json.load(json_file)
+
+
+class SeleniumTestCase(StaticLiveServerTestCase, TestConfig):
```

Comment:
pay attention! It's possible that since this class extends a  django test case class it will be detected as tests to run and will cause migrations to run and slow down the test process without any benefit.

Please double check this and if this is confirmed you could remove the inheritance on `StaticLiveServerTestCase` and rename this `SeleniumTestMixin`. Then you can inherit from ` `StaticLiveServerTestCase` in the actual test classes.

Reply:
You are right :+1: